mgr.go

```diff
@@ -832,14 +827,25 @@ func (mgr *SysboxMgr) getKernelHeaderSoftlink(rootfs string) (*configs.FsEntry,
 	}
 
 	// Create kernel-header softlink.
+	var fsEntries []configs.FsEntry
```

Comment:
Can you please move this "Create kernel-header softlink" comment further below where the corresponding code is being executed, and add a new comment here describing what we're doing? Something like ...

"In certain distros, such as 'alpine', the typical "/usr/src" kernel path may not exist, so create an associated fsEntry to ensure that the kernel softlink addition (below) can be properly carried out."